recipes/cuda-cuobjdump/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "cuda-cuobjdump" %}
+{% set version = "12.0.76" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_cuobjdump/{{ platform }}/cuda_cuobjdump-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: c2b627534d8eedff86b50bbf13c043a3c9a01a048fa6aa40678be5548c96d2a9  # [linux64]
+  sha256: 6b57dcc8a448068f0bd6a3c109f7dfbf96725f60c260a122a2c818f9cbe5b81c  # [ppc64le]
+  sha256: 30744ab5f7ae24d2265bd5167d750907abb2d2f6f62922aeab1fac5aa4030eca  # [aarch64]
+  sha256: 00688ad84b04df7e6a39e6d4d5fb29fda42d36bdf7e3a432c332ebafbed6fb90  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
```

Per https://github.com/conda-forge/cuda-version-feedstock/issues/1#issuecomment-1462998398, we should pin to `cuda-version`.
```suggestion
  build:
    - {{ compiler("c") }}
    - {{ compiler("cxx") }}
  run:
    - cuda-version =12.0
```